tests/functional/graphql/01-workflow.t

```diff
@@ -90,7 +90,7 @@ cat > expected << __HERE__
                 "title": "foo",
                 "description": "bar"
             },
-            "newestRunaheadCyclePoint": "1",
+            "newestRunaheadCyclePoint": "",
```

Comment:
What's the reason for this change?

Reply:
Apparently there's no runahead pool on start up, when now I initiate the pool earlier:
```
        if self.schd.pool.pool:
            pool_points = set(self.schd.pool.pool)
            workflow.oldest_cycle_point = str(min(pool_points))
            workflow.newest_cycle_point = str(max(pool_points))
        if self.schd.pool.runahead_pool:
            workflow.newest_runahead_cycle_point = str(
                max(set(self.schd.pool.runahead_pool))
            )
```
You can see this get's run, as the other's show up:
```
            "meta": {
                "title": "foo",
                "description": "bar"
            },
            "newestRunaheadCyclePoint": "",
            "newestCyclePoint": "1",
            "oldestCyclePoint": "1",
            "reloaded": false
```

Reply:
OK (I think) but maybe add a comment to that line in the test, because it looks weird.